event-sourcing/event-source/src/test/java/uk/gov/justice/services/eventsourcing/source/core/EnvelopeEventSourceTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.services.eventsourcing.source.core;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.justice.services.eventsourcing.publisher.core.EventPublisher;
+import uk.gov.justice.services.eventsourcing.repository.core.EventRepository;
+import uk.gov.justice.services.eventsourcing.stream.EventStream;
+
+import java.util.UUID;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EnvelopeEventSourceTest {
+
+    private static final UUID STREAM_ID = UUID.randomUUID();
+
+    @Mock
+    private EventRepository eventRepository;
+
+    @Mock
+    private EventPublisher eventPublisher;
+
+    @InjectMocks
+    private EnvelopeEventSource envelopeEventSource;
+
+    @Test
+    public void shouldReturnEventStream() {
+        EventStream eventStream = envelopeEventSource.getStreamById(STREAM_ID);
+
+        assertThat(eventStream.getId(), equalTo(STREAM_ID));
+    }
```

Comment:
Maybe we should test that the correct EventRepository and EventPublisher are set in the returned EventStream